app/src/main/java/com/example/cropshot/MainActivity.java

```diff
@@ -44,6 +44,7 @@
     ImageView imageView;
```

Comment:
We really need to deal with these global variables during the refactor @ef1739

Reply:
> We really need to deal with these global variables during the refactor @ef1739

My previous homework assignment's branch solved the global variable problem for the auto-cropping functions.